pandas/types/validate.py

```diff
@@ -0,0 +1,7 @@
+from pandas.types.common import is_bool
+
+def _validate_bool_kwarg(value, arg_name):
```

Comment:
move this to: https://github.com/pandas-dev/pandas/blob/master/pandas/util/validators.py

had forgotten that we put input validations here